source/distributed/fully_distributed_tria_util.cc

```diff
@@ -0,0 +1,520 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2019 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+#include <deal.II/base/geometry_info.h>
+#include <deal.II/base/mpi.h>
+
+#include <deal.II/distributed/fully_distributed_tria.h>
+#include <deal.II/distributed/fully_distributed_tria_util.h>
+#include <deal.II/distributed/tria.h>
+
+#include <deal.II/dofs/dof_accessor.h>
+#include <deal.II/dofs/dof_handler.h>
+
+DEAL_II_NAMESPACE_OPEN
+
+namespace parallel
+{
+  namespace fullydistributed
+  {
+    namespace Utilities
+    {
+      namespace
+      {
+        template <int dim, int spacedim>
+        void
+        set_flag_reverse(TriaIterator<CellAccessor<dim, spacedim>> cell)
```

Comment:
I can't figure out from the function name what the function does, and it doesn't have documentation :-)

```suggestion
        set_flag_reverse(const TriaIterator<CellAccessor<dim, spacedim>> &cell)
```